mobile/src/main/java/org/hawkular/client/android/fragment/FavMetricsFragment.java

```diff
@@ -183,9 +183,9 @@ private void tearDownBindings() {
     @Override
     public void onCreateOptionsMenu(Menu menu, MenuInflater menuInflater) {
         super.onCreateOptionsMenu(menu, menuInflater);
-        menuInflater.inflate(R.menu.search, menu);
+        menuInflater.inflate(R.menu.menu_search, menu);
 
-        MenuItem item = menu.findItem(R.id.menu_search);
+        MenuItem item = menu.findItem(R.id.menu_search1);
```

Comment:
Search was giving some errors. The menu in the action bar wasn't being populated with search querying.